test/geometry/arc.rb

```diff
@@ -2,24 +2,24 @@
 require 'geometry/arc'
 
 describe Geometry::Arc do
-    Arc = Geometry::Arc
+  Arc = Geometry::Arc
 
-    describe "when constructed" do
-	it "must accept a center point, radius, start and end angles" do
-	    arc = Geometry::Arc.new center:[1,2], radius:3, start:0, end:90
-	    arc.must_be_kind_of Geometry::Arc
-	    arc.center.must_equal Point[1,2]
-	    arc.radius.must_equal 3
-	    arc.start_angle.must_equal 0
-	    arc.end_angle.must_equal 90
-	end
+  describe "when constructed" do
+    it "must accept a center point, radius, start and end angles" do
+      arc = Geometry::Arc.new center:[1,2], radius:3, start:0, end:90
+      _(arc).must_be_kind_of Geometry::Arc
+      _(arc.center).must_equal Point[1,2]
+      _(arc.radius).must_equal 3
+      _(arc.start_angle).must_equal 0
+      _(arc.end_angle).must_equal 90
```

Comment:
Could we use the `assert_*` syntax instead? That syntax just gives me questions.